tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/PseudoLocales.java

```diff
@@ -284,6 +287,10 @@ protected CldrValue filter(CldrValue value) {
                 return defaultReturnValue;
             }
             String text = createMessage(value.getValue(), IS_PATTERN_PATH.test(path));
+                        if (GREGORIAN_SHORT_STANDARD_PATTERN.matches(path)) {
```

Comment:
please fix the indentation (are you using tabs? we only want spaces)

Reply:
Done (not clear what happened, tabs were not the issue).